taxonomies/ingredients.txt

```diff
@@ -52097,6 +52097,7 @@ fr:lentilles noires Beluga, lentilles Béluga
 # 8@2019-06-06
 
 <en:lentils
+de:braune Linsen
```

Comment:
```suggestion
en:brown lentils
de:braune Linsen
```

For ingredients, let's try to always have an English name.

Reply:
I modified it